plugins/catalog-backend/src/schema/openapi.yaml

```
@@ -72,6 +72,11 @@ components:
         type: array
         items:
           type: string
+      examples:
+        'Get name':
+          value: ['metadata.name']
```

```suggestion
        'Get name and the entire relations collection':
          value: ['metadata.name', 'relations']
```

just to have an example of getting complex fields, not just leaves